api/v1alpha4/types.go

```diff
@@ -106,11 +106,40 @@ type VnetSpec struct {
 	// +optional
 	CIDRBlocks []string `json:"cidrBlocks,omitempty"`
 
+	// Peerings defines a list of peerings of the newly created virtual network with existing virtual networks.
+	// +optional
+	Peerings VnetPeerings `json:"peerings,omitempty"`
+
 	// Tags is a collection of tags describing the resource.
 	// +optional
 	Tags Tags `json:"tags,omitempty"`
 }
 
+// Rename VnetPeeringSpec to VnetPeerings to remove the 'Spec'
+type VnetPeeringSpec struct {
+	// ResourceGroup is the resource group of the RemoteVnet.
+	ResourceGroup string `json:"resourceGroup"`
+
+	// RemoteVnetName defines name of the remote virtual network.
+	RemoteVnetName string `json:"remoteVnetName"`
+}
+
+// VnetPeerings is a slice of VnetPeering.
+type VnetPeerings []VnetPeeringSpec
+
+// type VnetPeeringSpec struct {
```

Comment:
Should this be deleted?

Reply:
To clarify, do you mean the type VnetPeerings? I added that in following the convention where a Subnets type was defined as an array of subnet specs. I removed the commented out struct below as well.